### DISCUSSION TITLE
After upgrading to rc60+, I'm getting "Unexpected error" when logging in

### DISCUSSION THREAD

**Discussion by teokk1** [directus/directus#5602] [Archive]
1) The issue and what you expected to happen
    I'm unable to login to Directus admin panel after upgrading from around rc40 to rc60 and newer.

2) Exact steps to reproduce this issue
   Upgrade Directus

3) Your environment:
    Directus rc64
    Sqlite
    Running on Ubuntu VPS
    Any browser

I'm certain I'm missing something, but since I've found absolutely no way to get any information other than "Unexpected error" and return code 400 in the logs, I have no idea what the problem is or how to fix it.

**Comment by paescuj** [comment]
Did you ran `npx directus database migrate:latest`?

See https://docs.directus.io/guides/projects/#upgrading-a-project

**Comment by teokk1** [comment]
Yes I did.

**Comment by rijkvanzanten** [comment]
> Exact steps to reproduce this issue
> Upgrade Directus

Could you clarify how exactly you upgraded Directus? Did you restart the node process after upgrading through npm?

**Comment by teokk1** [comment]
Yes, I restarted it. I also tried a multitude of methods and versions - using npm update, trying specific versions, deleting lockfiles and node_modules multiple times. 

When I create a new project with the same .env, it works fine. When I add a new account to the project with my current DB with `npx directus users create` the account gets added but I get the exact same error.

I've compared the databases of a new project and my current one and I can't find any significant difference that would cause this. I tried truncating all the metadata tables (directus_revisions, directus_sessions, etc.) in the hopes of fixing or at least simplyfing the issue but nothing works.

**Reply by rijkvanzanten** [reply]
What's the 400 error that's returned?
